public/css/freelancer-create.css

```diff
@@ -0,0 +1,141 @@
+/**
+ * Freelancer Creation View Styling
+ * js components in components/freelancer-creat.js
+ */
+
+
+/* freelancer create form */
+
+.freelancer-form {
+  margin: 10vh 10vw 5vh 10vw;
+  width: auto;
+  box-shadow: 0 1px 3px rgba(0,0,0,0.12), 0 1px 2px rgba(0,0,0,0.24);
+  border-radius: 2px;
+  background-color: #fff;
+}
+
+.freelancer-form-header {
+  padding: 20px;
+  color: #fff;
+  background-color: #F44336;
+  font-weight: 100;
+  border-radius: 2px 2px 0 0;
+  text-align: center;
+}
+
+.freelancer-form-header h1 {
+  font-weight: lighter;
+}
+
+.freelancer-form form {
+  display: flex;
+  flex-direction: column;
+  padding: 20px;
+}
+
+.names-input {
+  display: flex;
+  flex-direction: row;
+  margin-top: 20px;
+  margin-right: -20px;
+}
+
+.names-input .group {
+  margin-right: 20px;
+  width: 50%;
+}
+
+
+/* freelancer form label (courtesy of codeopen.io user )
+ * NOTE: possibly move this code to an external css file if form input
+ * fields will be used in other components outside of freelancer-create.js
+ */
+
+.group {
+  position: relative;
+  margin-bottom:45px;
```

Comment:
Also, tags are a bit too rounded considering that no other element on a job listing view page has rounded corners.